### PR TITLE
Change header menu API to dropdown menu to include v1 and v2

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by domcleal** [comment]
:+1: generally, thanks.  However the dropdown menu for API appears underneath Manual when I click it on Firefox, any idea why?

File: api/apidoc/v1.html [hunk 1]
```diff
@@ -0,0 +1,1100 @@
...
+    <div class="row">
+      <div id='container'>
+        <ul class='breadcrumb'>
+  <li class='active'><a href='../apidoc/v1.html'>Foreman v1</a></li>
+  <li class='pull-right'>
+    <a href="../apidoc/v1.html">v1</a> / <a href="../apidoc/v2.html">v2</a>
+  </li>
+</ul>
+
+<div><p>Foreman v1 is currently in the default API version.</p></div>
```
Comment (domcleal) [inline api/apidoc/v1.html]: "Foreman v1 is currently the default API version."

Does this need adding to core?

File: _includes/header.html [hunk 2]
```diff
@@ -59,6 +59,17 @@
                   {% endif %}
                   <li class="{{ active }}"><a href="{{ site.baseurl}}{{ link.url }}"> {{ link.text }} </a></li>
                   {% endfor %}
+
+                  <li class=""> <!-- fix active -->
+                    <a class="dropdown-toggle" id="dApi" data-toggle="dropdown" data-target="#" href="{{ site.baseurl}}api_v2.html">
+                      API
```
Comment (isratrade) [inline _includes/header.html:66]: @abenari, @skottler, do you have any idea why the dropdown menu floats to the far left under "Manual"

File: api/apidoc/v1.html (hunk 1 above)
Comment (isratrade) [inline api/apidoc/v1.html]: @domcleal, you are correct.  This text is in Foreman core and it is generated automatically in the docs.  Here's the PR
https://github.com/theforeman/foreman/pull/1092

**Comment by isratrade** [comment]
I haven't looked closely at all the example data.  However, I think it can be merged (after the drop-down is fixed) and we can fixed examples one-at-a-time as necessary.

**Comment by isratrade** [comment]
@domcleal, where's a good place to document how to update the documentation (a wiki for the wiki "-).  Basic steps are:

cd to foreman directory
1.  rake apipie:cache

cd to theforeman.org directory
2.  cp -r dir/to/foreman/public/apipie-cache/apidoc api/
3.  cp dir/to/foreman/public/apipie-cache/apidoc.\* api/

**Comment by domcleal** [comment]
@isratrade add it to README.md

**Comment by isratrade** [comment]
@domcleal, re-committed with additions to README.md

**Comment by isratrade** [comment]
@domcleal, API dropdown menu fixed and re-committed.

**Comment by domcleal** [comment]
Really nice job, thanks @isratrade!

**Pull request merged**